### PR TITLE
Added HOP New Bridge Methods And New Event

### PR DESCRIPTION
**Pull request by kelvinator07**
- Methods:
  getEstimatedFeesForNextPegOutEvent()
  getNextPegoutCreationBlockNumber()
  getQueuedPegoutsCount()

- Event:
  batch_pegout_created

### CODE REVIEW

**Review by marcos-iov — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rsk-precompiled-abis",
-  "version": "4.0.2-IRIS",
+  "version": "4.0.3",
```
Comment (josedahlquist) [inline package.json:3]: Please update the readme including the hop version in its compatibility matrix
```suggestion
  "version": "5.0.0-HOP",
```

**Review by josedahlquist — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kelvinator07) [reply package.json]: Resolved

**Review by nathanieliov — COMMENTED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (nathanieliov) [reply package.json]: Should not the "Important note" section be updated too? I guess ~~ IRIS ~~ should be updated to HOP.

Currently text:
```
# Important note
If the version to be installed is not defined in the command line, the version will correspond to the latest version in IRIS.
```

Comment (kelvinator07) [reply package.json]: Good Catch Nath. Updated To HOP

**Review by nathanieliov — APPROVED** [review]

**Review by marcos-iov — DISMISSED** [review]

**Review by josedahlquist — CHANGES_REQUESTED** [review]
please run `npm install` on your local env so it updates package-lock.json file as well 🙏

File: package-lock.json [hunk 2]
```diff
@@ -1,4908 +1,8 @@
 {
   "name": "@rsksmart/rsk-precompiled-abis",
-  "version": "4.0.2-IRIS",
-  "lockfileVersion": 2,
```
Comment (marcos-iov) [inline package-lock.json:4]: Check your version of node. It should be v16

Comment (kelvinator07) [reply package-lock.json:4]: Done

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**